apps/deploy-web/src/components/providers/ProviderRawData.tsx

```diff
@@ -11,11 +11,19 @@ import Layout from "../layout/Layout";
 import { CustomNextSeo } from "../shared/CustomNextSeo";
 import ProviderDetailLayout, { ProviderDetailTabs } from "./ProviderDetailLayout";
 
+export const COMPONENTS = {
+  Layout,
+  CustomNextSeo,
+  ProviderDetailLayout,
+  DynamicReactJson
+};
```

Comment:
this is added for easier mocking of component dependencies in unit tests. Follows similar pattern as Vue/Angular component dependencies